waspc/src/Wasp/Generator/DbGenerator/Jobs.hs

```diff
@@ -178,9 +178,12 @@ runPrismaCommandAsJobWithExtraEnv ::
   [String] ->
   J.Job
 runPrismaCommandAsJobWithExtraEnv fromDir envVars projectRootDir cmdArgs =
-  runNodeCommandAsJobWithExtraEnv envVars fromDir (absPrismaExecutableFp waspProjectDir) cmdArgs J.Db
+  runNodeCommandAsJobWithExtraEnv (defaultEnvVars ++ envVars) fromDir (absPrismaExecutableFp waspProjectDir) cmdArgs J.Db
   where
     waspProjectDir = projectRootDir </> waspProjectDirFromProjectRootDir
+    defaultEnvVars =
+      [ ("PRISMA_HIDE_UPDATE_MESSAGE", "true")
```

Comment:
All calls happen through this function since calling Prisma is not as simple as `npx prisma` due to a Prisma bug with resolving the schema file (or something along those lines, but that bug forces us to call prisma using `node ./node_modules/.../prisma`)

Reply:
Ok that is good then! I am still curious, any thoughts on the potentially different solution, that I mentioned above?

Reply:
If we set some variable _globally_, won't this affect all other command calls? You never know if some other tool in the future starts checking for some env var in the future. If we do it like this, we have isolated the effect to this exact command. 

We can use `setEnv` from `System.Environment` to set it and I've tested it by running `env` as command to see if the env var is picked up, and it is. So it's a viable option, but I'm not sure if we should do it 🤷

Reply:
Oh ok nice! If that is so, I would consider going for that, because we really don't want any command to warn user about updating Prisma, right? This way, we know that warning won't come up for sure (and I think that is ok), instead of it coming up if we call prisma in any other way or call it somehow indirectly or whatever. No/less place for future mistake on our side.

Aha but wait, the question is: does this set up that env var count only for the process and subprocesses or `wasp`, or does it actually affect the terminal session in which `wasp` CLI is running and leave it like that for the following commands after it? If the latter is correct, then no, it is not an option, but if the first is correct, then I would say let's go for it.

Reply:
What about this argument?

> If we set some variable globally, won't this affect all other command calls? You never know if some other tool in the future starts checking for some env var in the future.

Let me test this: 

> does it actually affect the terminal session in which wasp CLI is running and leave it like that for the following commands after it

Reply:
Setting the env var using `setEnv` doesn't affect the terminal session it seems. I've tested it like this:
1. Run `PRISMA_HIDE_UPDATE_MESSAGE=foo`
2. Run `wasp version` (which runs `setEnv "PRISMA_HIDE_UPDATE_MESSAGE" "true")
3. Run `echo $PRISMA_HIDE_UPDATE_MESSAGE`
4. Which outputs `foo`

Reply:
> If we set some variable globally, won't this affect all other command calls? You never know if some other tool in the future starts checking for some env var in the future.

Sure, that's a good point, but this is very specific env var, and if some other command will indeed check it, it will be doing so in the context of `wasp cli`, and then we do want it to see it set to "true", so I would say this is perfect, we want this.